app/views/ncr/work_orders/_ba_80_fields.html.haml

```diff
@@ -1,6 +1,7 @@
 .form-group{data: {filter_key: 'expense-type', filter_value: 'BA80'}}
   = f.label :rwa_number, class: "required"
-  = f.text_field :rwa_number, class: 'form-control'
+  = f.text_field :rwa_number, class: 'form-control', 'data-parsley-conditionalrequired':'#ncr_work_order_expense_type_ba80', 'data-parsley-validate-if-empty':'true', 'data-parsley-conditionalrequired-message':'RWA Number is required for BA80 expense types'
```

Comment:
Maybe we have the field required, but only if visible? That seems like a general-purpose solution